src/bufr/BufrParser/Query/Constants.h

```diff
@@ -9,6 +9,6 @@
 namespace Ingester {
 namespace bufr {
     /// \brief The missing data value for all BUFR data.
-    const float MissingValue = 10e10;
+    const double MissingValue = 10.0e10;
```

Comment:
Not sure it matters, but would this make more sense as 10.0d10 since it is declared double precision?
I also admit some confusion because I thought BUFR had a single-precision missing value since a double missing would require more space to store than a single-precision.  Is this actually a NCEP bufr-lib thing not actually something inherent to BUFR storage itself?